crates/router/src/connector/dummyconnector.rs

```diff
@@ -21,32 +21,33 @@ use crate::{
 };
 
 #[derive(Debug, Clone)]
-pub struct DummyConnector;
-
-impl api::Payment for DummyConnector {}
-impl api::PaymentSession for DummyConnector {}
-impl api::ConnectorAccessToken for DummyConnector {}
-impl api::PreVerify for DummyConnector {}
-impl api::PaymentAuthorize for DummyConnector {}
-impl api::PaymentSync for DummyConnector {}
-impl api::PaymentCapture for DummyConnector {}
-impl api::PaymentVoid for DummyConnector {}
-impl api::Refund for DummyConnector {}
-impl api::RefundExecute for DummyConnector {}
-impl api::RefundSync for DummyConnector {}
-impl api::PaymentToken for DummyConnector {}
-
-impl
+pub struct DummyConnector<const T: i32>;
```

Comment:
You could also have this as a `u8` instead, since I don't expect the number of dummy connectors to exceed 255.